File: BookMyShowAPI/DomainModels/Theatre.cs
namespace DomainModels
{
    public class Theatre
    {
        public int Id { get; set; }
        public string? TheatreName { get; set; }
        public int? TheatreRows { get; set; }
        public int? TheatreColumns { get; set; }
        public List<int>? MovieIds { get; set; }
        public List<TimeOnly>? MovieTimings { get; set; }
        public string? LocationName { get; set; }
        public float TicketPrice { get; set; }
    }
}

File: BookMyshowFrontend/bookMyShow/src/app/shared/models/theatre.ts
import { Time } from "@angular/common";

export class Theatre {
    id: number;
    theatreName: string;
    theatreRows: number;
    theatreColumns: number;
    movieIds: number[];
    movieTimings: Time[];
    locationName: string;
    ticketPrice: number;
    constructor(id: number, theatreName: string, locationName: string, movieIds: number[], theatreRows: number, theatreColumns: number, movieTimings: Time[], ticketPrice: number) {
        this.id = id;
        this.theatreName = theatreName;
        this.theatreRows = theatreRows,
            this.theatreColumns = theatreColumns,
            this.movieTimings = movieTimings,
            this.ticketPrice = ticketPrice,
            this.movieIds = movieIds,
            this.locationName = locationName
    }
}